CatsAndDogs/ContentLoader.swift

```diff
@@ -16,11 +16,38 @@ private extension Data {
 }
 
 class OnlineContentLoader {
-    let url = URL(string: "https://catsanddogs-kotlin-bff.herokuapp.com/schedule.json?from=2017-05-08T12:00:00-02:00")
-    let session = URLSession(configuration: .default)
+    private let urlString = "https://catsanddogs-kotlin-bff.herokuapp.com/kotlinconf/schedule.json"
+    private let session = URLSession(configuration: .default)
 
     private func downloadSchedule(completion: @escaping (Data?)->()) {
-        session.dataTask(with: url!) { (data, _, _) in completion(data) }.resume()
+        let urlRequest: URLRequest? = {
+            guard var urlComponents = URLComponents(string: self.urlString) else { return nil }
+            
+            let currentLocale = Locale.US
+            let currentTimeZone = TimeZone.California!
+            
+            let dateFormatter = RFC3339DateFormatter(locale: currentLocale,
+                                                     timezone: currentTimeZone)
+  
+            let dateConferenceStarts = dateFormatter.date(from: "2017-11-02T11:00:00-07:00")
```

Comment:
On Android, we have a date time picker. Maybe we should align functionalities across platforms.

Reply:
We should probably add pull to refresh for both apps so we don't need a picker, but the apps are always the most accurate?